source/__tests__/integration/create-property.test.js

```diff
@@ -25,7 +25,7 @@ describe('PropertyTemplate requirements', () => {
     it('clicking "Add Property Template" appends a property template section to the form', async () => {
       expect.assertions(1)
       await page.waitForSelector(ptFieldsTableSel)
-      await pupExpect.expectSelToExist('i.fa-exclamation[id="error"]')
+      await expect(page).toMatchElement('i.fa-exclamation[id="error"]')
```

Comment:
toMatchElement() is from jest-puppeteer and passes and fails like the method I wrote ... so I got rid of mine.